src/coreComponents/meshUtilities/InternalMeshGenerator.hpp

```diff
@@ -85,17 +42,44 @@ class InternalMeshGenerator : public MeshGeneratorBase
   InternalMeshGenerator( const string & name,
                          Group * const parent );
 
-  virtual ~InternalMeshGenerator() override;
+  virtual ~InternalMeshGenerator() override = default;
 
   /**
    * @brief Return the name of the InternalMeshGenerator in object Catalog.
    * @return string that contains the key name to InternalMeshGenerator in the Catalog
    */
   static string catalogName() { return "InternalMesh"; }
 
-//  void ProcessInputFile( xmlWrapper::xmlNode const & targetNode ) override;
-//
-//
+  ///@cond DO_NOT_DOCUMENT
+  struct viewKeyStruct
+  {
+    constexpr static char const * xCoordsString() { return "xCoords"; }
+    constexpr static char const * yCoordsString() { return "yCoords"; }
+    constexpr static char const * zCoordsString() { return "zCoords"; }
+    constexpr static char const * xElemsString() { return "nx"; }
+    constexpr static char const * yElemsString() { return "ny"; }
+    constexpr static char const * zElemsString() { return "nz"; }
+    constexpr static char const * xBiasString() { return "xBias"; }
+    constexpr static char const * yBiasString() { return "yBias"; }
+    constexpr static char const * zBiasString() { return "zBias"; }
+    constexpr static char const * cellBlockNamesString() { return "cellBlockNames"; }
+    constexpr static char const * elementTypesString() { return "elementTypes"; }
+    constexpr static char const * trianglePatternString() { return "trianglePattern"; }
+    constexpr static char const * meshTypeString() { return "meshType"; }
+  };
+  /// @endcond
+
+  /**
+   * @enum MeshType
+   *
+   * The options for mesh type
+   */
+  enum class MeshType : integer
+  {
+    Cartesian,
+    Cylindrical,
+    CylindricalSquareBoundary
+  };
```

Comment:
I like this since it brings clear view of possibilities and adds more readability.